script/app-coverage-report.js

```diff
@@ -3,18 +3,32 @@
 const fs = require('fs');
 const path = require('path');
 const Table = require('cli-table');
+const {
+  getAppManifests,
+} = require('../src/site/stages/build/webpack/manifest-helpers');
 
 const printCoverage = coverageResults => {
   // Create a new table with headers
   const coverageTable = new Table({
-    head: ['Application', 'Lines', 'Functions', 'Statements', 'Branches'],
+    head: [
+      'Application (src/applications)',
+      'Lines',
+      'Functions',
+      'Statements',
+      'Branches',
+    ],
   });
 
   // Add each app coverage result to the table
   Object.keys(coverageResults).forEach(key => {
     if (key !== 'total') {
+      const appLocation = `${coverageResults[key].path.substr(
+        0,
+        coverageResults[key].path.lastIndexOf('/'),
+      )}`;
+
       coverageTable.push({
-        [key]: [
+        [appLocation]: [
           `${coverageResults[key].lines.pct}%`,
           `${coverageResults[key].functions.pct}%`,
           `${coverageResults[key].statements.pct}%`,
```

Comment:
It looks like you might be able to just loop through the `values` instead since that is what the key is being used for:

```js
  // Add each app coverage result to the table
  Object.values(coverageResults).forEach(cov => {
    const appLocation = `${cov.path.substr(0, cov.path.lastIndexOf('/'))}`;

    coverageTable.push({
      [appLocation]: [
        `${cov.lines.pct}%`,
        `${cov.functions.pct}%`,
        `${cov.statements.pct}%`,
        `${cov.branches.pct}%`,
      ],
    });
  });
```